components/org.wso2.carbon.identity.application.authenticator.google/src/main/java/org/wso2/carbon/identity/application/authenticator/google/GoogleOAuth2Authenticator.java

```diff
@@ -58,6 +70,158 @@ private void initTokenEndpoint() {
         }
     }
 
+    @Override
+    public boolean canHandle(HttpServletRequest request) {
+
+        boolean canHandle = false;
+        if (log.isDebugEnabled()) {
+            log.debug("Inside OpenIDConnectAuthenticator.canHandle()");
+        }
+        if (StringUtils.isNotBlank(request.getParameter("oneTapEnabled")) &&
```

Comment:
We have used this `oneTapEnabled` in several places. Can we introduce a constant for this? WDYT?

Reply:
Fixed f2aa0c1666f9b5234e7d113125fadd55008081cd